packages/stores/src/account.ts

```diff
@@ -1,18 +1,33 @@
-import { Logger, WalletManager } from "@cosmos-kit/core";
+import { StdFee } from "@cosmjs/launchpad";
```

Comment:
FYI I think this package is deprecated

https://github.com/chainapsis/keplr-wallet/issues/521

Reply:
Thanks for pointing this out! Using `@cosmjs/stargate` now